tests/conftest.py

```diff
@@ -346,6 +362,58 @@ def persisted_transfer_carol_car_to_dimi(carol_keypair, dimi_pubkey, bdb_node,
 
 
 @fixture
+def persisted_transfer_dimi_car_to_ewy(dimi_keypair, ewy_pubkey, bdb_node,
+                                       persisted_transfer_carol_car_to_dimi):
+    # FIXME The sleep, or some other approach is required to wait for the
+    # transaction to be available as some processing is being done by the
+    # server.
+    sleep(1.5)
```

Comment:
Maybe these sleeps should happen at the end of each fixture rather than requiring the user of the fixture to add them manually?